modules/gcloud/src/test/java/org/testcontainers/containers/PubSubEmulatorContainerTest.java

```diff
@@ -0,0 +1,72 @@
+package org.testcontainers.containers;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+import com.google.api.core.ApiFuture;
+import com.google.api.core.ApiFutureCallback;
+import com.google.api.core.ApiFutures;
+import com.google.api.gax.core.NoCredentialsProvider;
+import com.google.api.gax.grpc.GrpcTransportChannel;
+import com.google.api.gax.rpc.FixedTransportChannelProvider;
+import com.google.api.gax.rpc.TransportChannelProvider;
+import com.google.cloud.pubsub.v1.Publisher;
+import com.google.cloud.pubsub.v1.TopicAdminClient;
+import com.google.cloud.pubsub.v1.TopicAdminSettings;
+import com.google.common.util.concurrent.MoreExecutors;
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.PubsubMessage;
+import com.google.pubsub.v1.Topic;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PubSubEmulatorContainerTest {
+
+    @Rule
+    public PubSubEmulatorContainer emulator = new PubSubEmulatorContainer();
+
+    @Test
+    public void testSimple() throws IOException, ExecutionException, InterruptedException {
```

Comment:
It would be nice to have a(nother) test that:
1. Publishes a message with some trivial contents
1. Pulls that message using from a subscription to that topic
1. Verifies that the trivial contents was received

Reply:
@walles thanks for the suggestions. I have added one more commit introducing that change. 👍🏽

Reply:
Beautiful! 🥇